service-fabrik-plugin.go

```diff
@@ -68,12 +69,12 @@ func (serviceFabrikPlugin *ServiceFabrikPlugin) Run(cliConnection plugin.CliConn
 	}
 
 	var cmds []string = strings.Split(args[0], "-")
-	//2 overall switches: backup & restore
+	//2 overall switches: backup, restore & events
```

Comment:
**3** overall switches

Reply:
Also add in (below comment)[https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-3a9c19630bb59355ffe1ec2155830875L106] new flags.